docs/en/status-communities/delegate-admin-functions-to-your-community.md

```diff
@@ -0,0 +1,60 @@
+---
+id: 543
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Delegate admin functions to your community
+
+!!! note ""
+    Currently, you can only delegate admin functions using Status desktop. [:octicons-desktop-download-16: Download Status desktop][status-web-download].
+
+Community owners have total control over their communities, including settings, membership approvals and permissions. In addition, owners can delegate administrative functions to other community members through the [admin role][about-the-owner-and-admin-community-roles].
+
+To delegate admin functions to others, owners use [token-based community permissions][set-up-your-community-permissions]. Tokens are digital assets in a blockchain (such as the [Ethereum blockchain][about-the-ethereum-blockchain]), including assets (virtual coins), collectibles and ENS names.
+
+## What to expect
+
+- To delegate admin functions, you assign the `Become admin` community permission to a collectible or ENS name owner.
+- Using ENS names to delegate admin functions is more straightforward than using collectibles, but it requires the admin to own the ENS name.
+- You can create a maximum of five different `Become admin` community-level permissions, but you can apply a single permission to multiple members.
+- To stop delegating admin functions, [self-destruct the admin token][self-destruct-a-token-remotely] or [delete the admin permission][delete-community-permissions].
+
+## How to delegate admin functions
```

Comment:
This title “How to delegate admin functions” doesn't look distinguishable enough from the title for the procedure below. I'd change it to something like "Choose a token for delegation".
For the content in this section, using a table could be more straightforward for readers to see the difference between these tokens. And I'd put the content in the call-out in the table as well. For example:
| ✓ Collectibles | ✓ ENS names | x Assets |
|----|---|----|
|bla|bla|bla|

Reply:
Good point. I love the idea of using a table, but after implementing this approach, it didn't look that good :/ 

I also tried to use bullet points, as you suggested, but the content still doesn't look as good as the first version, IMHO.

This content is not suitable for tables or bullet points, I guess. Therefore, I'll keep the content as it is. 

![Screenshot 2023-04-12 at 9 22 01@2x](https://user-images.githubusercontent.com/62282406/231383428-f169b01e-4360-4a7c-b03f-144b775f8ae3.png)
![Screenshot 2023-04-12 at 9 21 30@2x](https://user-images.githubusercontent.com/62282406/231383429-cb30752f-a15b-48e0-900e-7d657a1a8ed6.png)
![Screenshot 2023-04-12 at 9 51 52@2x](https://user-images.githubusercontent.com/62282406/231390679-920df2dd-adb0-47f3-84fb-eca519b0cbe7.png)